sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -52,13 +54,15 @@
 - Moved `GetAlertConfigurationsOptions`, `GetDatasourceCredentialsOptions`, and `GetDetectionConfigurationsOptions` to the `Azure.AI.MetricsAdvisor.Administration` namespace.
 - Moved `DatasourceCredential`, `DataFeedSource`, `NotificationHook`, and all of their concrete child types to the `Azure.AI.MetricsAdvisor.Administration` namespace.
 - Moved `MetricFeedback` and all of its concrete child types to the `Azure.AI.MetricsAdvisor` namespace.
+- In `GetAllFeedbackOptions`, moved all feedback filter properties to a new nested property `Filter` of type `GetAllFeedbackFilter`.
```

Comment:
Basically we had all properties flattened in `GetAllFeedbackOptions`. We moved some of them to `GetAllFeedbackFilter`, and made it a new property of the options type. We're only doing this for cross-language consistency, but we have a similar pattern in other types as well (`GetDetectionConfigurationsOptions` and `GetDataFeedsOptions`).